auth-tokens/src/main/java/com/palantir/tokens/auth/BearerTokens.java

```diff
@@ -78,7 +78,7 @@ public static BearerToken fromPath(Path path) throws IOException {
         }
 
         LOGGER.warn("No token file found in any of the configured paths: {}.", paths);
-        return Optional.absent();
+        return Optional.empty();
```

Comment:
another idea is to just not remove the guava dependency. Optional has been in Guava since 10.0 and there are no plans to even _deprecate_ guava optional 'for the foreseeable future'.

Given that the caller who wishes a Java optional can just use Guava 21+ and just call .toJavaOptional(), we might as well avoid the break entirely.